test/server/cards/14.4-CoC/StoicRival.spec.js

```diff
@@ -0,0 +1,51 @@
+describe('Stoic Rival', function() {
+    integration(function() {
+        describe('Stoic Rival\'s ability', function() {
+            beforeEach(function() {
+                this.setupTest({
+                    phase: 'conflict',
+                    player1: {
+                        inPlay: ['stoic-rival'],
+                        hand: ['vine-tattoo','ornate-fan']
+                    },
+                    player2: {
+                        inPlay: ['shiba-tsukune', 'otomo-courtier']
+						hand:['ornate-fan','ornate-fan','ornate-fan']
+                    }
+                });
+				this.initiateConflict({
+                    attackers: ['stoic-rival'],
+                    defenders: ['shiba-tsukune','otomo-courtier']
+                });
+                this.noMoreActions();
```

Comment:
Your flow is backwards here.  phase: [phasename] essentially forwards the game to the start of the phase.  If you put 'draw' here, for example, you'll need to make both players bid for cards as your first step.

Since you start it at conflict, player1 is currently being presented with their pre-conflict window.
"InitiateConflict" is a wrapper for "Click Ring, Click Attackers, Click Province, click "Start Conflict", click Defenders, click Done".  You're not at that step yet, as both players need to pass before a conflict can be declared.  That's what noMoreActions does (it's dumb and just calls .pass for both players).

Reply:
Thanks, moving around noMoreActions was inadvertent but I clearly wasn't completely aware of what was going on!